app/src/main/res/layout/activity_pin.xml

```diff
@@ -94,8 +92,8 @@
         android:layout_marginBottom="8dp"
         android:orientation="horizontal"
         app:layout_constraintBottom_toTopOf="@+id/brkeyboard"
-        app:layout_constraintLeft_toLeftOf="parent"
-        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
```

Comment:
So is this so whether it is RTL or LTR languages it auto centers @andhikayuana . I wonder, is this good enought?

Reply:
Yes, the layout auto-centers correctly for both RTL and LTR languages. I’ve tested it, and it seems to work well (self test)
more detail: https://developer.android.com/training/basics/supporting-devices/languages#MirroringUpdateResources